extensions/ql-vscode/src/view/variant-analysis/VariantAnalysisRepositoriesStats.tsx

```diff
@@ -68,7 +68,7 @@ export const VariantAnalysisRepositoriesStats = ({
   if (variantAnalysisStatus === VariantAnalysisStatus.Failed) {
     return (
       <>
-        0<HorizontalSpace size={2} />
```

Comment:
I'm not sure the `0` was a bug. Or if we do want to remove it, do we need to also remove some whitespace before the icon?

It might be simpler regardless to leave it for this PR and address as a separate PR. What do you think?

Screenshot before:
![Screenshot 2023-11-02 at 16 10 34](https://github.com/github/vscode-codeql/assets/3749000/f3dc3771-5d70-447d-a7f2-29fbfbf0e238)

Screenshot after:
![Screenshot 2023-11-02 at 16 10 48](https://github.com/github/vscode-codeql/assets/3749000/7f1cb6cc-42da-409e-b743-8c85a9c95499)

Reply:
I think you're right. This just looked like an accidental typo, but I didn't actually check whether that was the case.

In this case, I would have expected something like `0 <ErrorIcon label="Variant analysis failed" />` rather than a separate element for the spacing.